release.sh

```diff
@@ -145,21 +153,43 @@ if [ $dodist -eq 0 ]; then
     # We haven't tagged yet, so tell the dist script what version
     # it's building
     DIST_VERSION="$tag" npm run dist
+
+    `dirname $0`/scripts/changelog_head.py > latest_changes.md
```

Comment:
The release script is common to js-sdk and react-sdk though, and this would all be riot-web specific logic to insert js-sdk and react-sdk changelogs.